dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -187,11 +187,12 @@ def level_properties
 
     @level = @script_level.level
 
-    @video = @level.specified_autoplay_video&.summarize(false)
+    @video = @level.specified_autoplay_video&.summarize(false)&.camelize_keys
 
-    @merged_properties = @level.properties.camelize_keys.merge({video: @video&.camelize_keys})
+    @properties = @level.properties.camelize_keys
+    @properties[:levelData] = @video if @video
```

Comment:
I was also thinking about this for a separate change; as this gets more complex, should we just add a `summarize_for_lab2` function on `level.rb` where we can craft the structure of the payload we want to send, and move the logic out of the controller?

Reply:
Oh, that is a good idea, and definitely the right path forward.

Reply:
Added a `summarize_for_labs2_properties`, though flexible on naming.